Exercise_1.java

```diff
@@ -6,28 +6,46 @@
     boolean isEmpty() 

```

Comment:
Good Job! Please add some comments and time and space complexities for all the stack operations